cmd/kube-controller-manager/app/autoscaling.go

```diff
@@ -92,7 +92,6 @@ func startHPAControllerWithMetricsClient(ctx ControllerContext, metricsClient me
 		replicaCalc,
 		ctx.InformerFactory.Autoscaling().V1().HorizontalPodAutoscalers(),
 		ctx.ComponentConfig.HPAController.HorizontalPodAutoscalerSyncPeriod.Duration,
-		ctx.ComponentConfig.HPAController.HorizontalPodAutoscalerUpscaleForbiddenWindow.Duration,
 		ctx.ComponentConfig.HPAController.HorizontalPodAutoscalerDownscaleForbiddenWindow.Duration,
```

Comment:
why do we still have the downscale forbidden window?  We should think about how to get rid of these together

Reply:
I really would rather do it in a separate PR:
- This PR will be really big if I add one more change to it
- I'd rather talk to @mwielgus about it before getting rid of the downscale forbidden window